packages/website/tsconfig.json

```diff
@@ -9,6 +9,8 @@
         "strictNullChecks": false,
         "noImplicitThis": false,
         "declaration": false,
+        "declarationMap": false,
```

Comment:
These are overrides for the new settings in the base __tsconfig.json__ file. Website is the only package that does not support project references right now. Project references requires `allowJs: false` and website requires `allowJs: true`.

Reply:
@albrow can you add this commend in the JSON file with a hack?